### PR TITLE
Feature/create bugfix

### PR DESCRIPTION
**Pull request by megellman**

### CODE REVIEW

File: public/js/createEntry.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
         if(response.ok){
-            document.location.replace('/journal');
+            document.location.replace(`../../journals/${id}`);
         } else {
             alert('Failed to create entry');
         }
     }
 };
 
-document.querySelector('#newEntry').addEventListener('submit', entryForm);
```
Comment (rmoscoe) [inline public/js/createEntry.js:23]: This is actually the correct version.

**Review by rmoscoe — CHANGES_REQUESTED** [review]
One change requested.

File: public/js/createEntry.js (hunk 1 above)
Comment (megellman) [reply public/js/createEntry.js:23]: that newEntry id doesn't exist

Comment (megellman) [reply public/js/createEntry.js:23]: Nevermind. I just checked and there are two IDs. The one you're referring to that's on the form and the one I used here that's on the button of the form. I just removed the form id so there's no more confusion

**Review by rmoscoe — APPROVED** [review]

**Pull request merged**